src/types/api/models/disputes.ts

```diff
@@ -1,6 +1,8 @@
 import { Schema } from '../utils';
-import { components } from '../resources/DisputesResource';
+import { components, operations } from '../resources/DisputesResource';
 
 export type IDispute = Schema<components, 'Dispute'>;
+export type IDisputeListItem = Schema<components, 'DisputeListItem'>;
 export type IDisputeDetail = Schema<components, 'DisputeDetailResponse'>;
+export type IDisputeStatusGroup = operations['getDisputeList']['parameters']['query']['statusGroup'];
```

Comment:
We shouldn't have to rely on `operations` from the schema to derive `IDisputeStatusGroup`.
```suggestion
import { components } from '../resources/DisputesResource';

export type IDispute = Schema<components, 'Dispute'>;
export type IDisputeListItem = Schema<components, 'DisputeListItem'>;
export type IDisputeDetail = Schema<components, 'DisputeDetailResponse'>;
export type IDisputeStatusGroup = Schema<components, 'StatusGroup'>;
```

Reply:
✅